vignettes/advanced_chunks.Rmd

```diff
@@ -9,6 +9,8 @@ vignette: >
   %\VignetteEncoding{UTF-8}
 ---
 
+NOTE: Chunks are being deprecated `qenv` objects should be used instead - new vignettes are on their way.
```

Comment:
I would think about making these notes stand out a bit more. Maybe make 'NOTE' bold.

Reply:
True - although before we merge into main the plan is to remove these vignettes completely and write new ones for qenv -> see [here](https://github.com/insightsengineering/teal/issues/731) 

![image](https://user-images.githubusercontent.com/15201933/191694097-27b259f8-6685-44a7-870a-5dfecb1d0ef2.png)

So this was a temporary fix to make the teal_refactor branch sort of correct